packages/components/src/disabled/index.js

```diff
@@ -31,40 +36,11 @@ const DISABLED_ELIGIBLE_NODE_NAMES = [
 	'TEXTAREA',
 ];
 
-class Disabled extends Component {
-	constructor() {
-		super( ...arguments );
+function Disabled( { className, children, ...props } ) {
+	const node = useRef( null );
```

Comment:
```suggestion
	const node = useRef();
```

I don't think it makes any difference but was wondering why setting it explicitly here.

Reply:
I don't think it made any difference.